CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ CHANGELOG
 
 #### Monitor
 * Fix link to monitor page (#157).
+* Parameters panel that displays bot parameters. If there is a file or a directory from within the allowed path (specified in configs.php), the contents gets displayed so the administrator is aware about the configuration the bot has.
```

Comment:
I needed some time to get the meaning of the allowed paths. Proposal:
> The new parameters panel that displays the bot's parameters on the bot's monitor page. For parameters which are filenames, the content of these files will be shown as text if they reside in the allowed paths (set in config.php).

Reply:
fully agree